library.properties

```diff
@@ -0,0 +1,9 @@
+name=SPBTLE-RF
+version=1.0.0
+author=Wi6Labs
+maintainer=STMicroelectronics
+sentence=This library includes drivers for ST's BlueNRG/BlueNRG-MS Bluetooth Low Energy device.
```

Comment:
may be interesting to add BLE somewhere for search feature in the library manager ?